plugin/rewrite/README.md

```diff
@@ -180,7 +180,7 @@ rewrite [continue|stop] name regex STRING STRING answer name STRING STRING
 Using FIELD edns0, you can set, append, or replace specific EDNS0 options on the request.
 
 * `replace` will modify any matching (what that means may vary based on EDNS0 type) option with the specified option
-* `append` will add the option regardless of what options already exist
+* `append` will add the option if does not find any matching option
```

Comment:
I find both the old and new ambiguous. `append` as the EDNS option regardless of what option are already in the section? I.e. you can get duplicates? Is that what this is trying to say?

Reply:
do you mean...

`append` will add the option if it does not already exist

Reply:
or perhaps even more precise...

`append` will add the option *only* if it does not already exist

Reply:
@miekg : what is ambiguus in the new definition ?
@chrisohaver : is used in purpose 'matching' because it is defined in 'replace'.
=> (what that means may vary based on EDNS0 type)

we are doing the same 'matching' test for all those operations.

Reply:
OK, how about...

`append` will add the option if no matching option exists

Reply:
also, IMO, the way that "matching" is defined is awkward and disrupts the sentence.  It would be better to say something like ...

 * `replace` will modify any "matching" option with the specified option.  The criteria for "matching" varies based on EDNS0 type.

Reply:
@chrisohaver : Thanks. I will update it.